src/core/openthread-core-default-config.h

```diff
@@ -306,6 +306,16 @@
 #endif  // OPENTHREAD_CONFIG_MAX_JOINER_ENTRIES
 
 /**
+ * @def OPENTHREAD_CONFIG_MAX_JOINER_ENTRIES
+ *
+ * The maximum number of Joiner Router entries that can be queued by the Joiner.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_MAX_JOINER_ROUTER_ENTRIES
+#define OPENTHREAD_CONFIG_MAX_JOINER_ROUTER_ENTRIES                    2
```

Comment:
this `2` not aligned with other definition.

Reply:
fixed